portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +24,16 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  /**
```

Comment:
Typically you do not need to add a method JSDoc for overriden methods. I would keep the general description because it mentions responding to Hello Amy. The @param and @throws do not seem necessary though given it is the same as the method it overrides.

Reply:
I just wanted to make it clear that adding @params and @throws is really good, just in the case of @Override it is best not too. 

One of the main reasons we do not add JsDoc for @Override is because if the method changes functionality and you want to update the doc, then you need to update it for every method overriding it. So it can become annoying to do.

Reply:
Understood!

Done!